src/app/@theme/components/header/header.component.ts

```diff
@@ -11,7 +11,7 @@ import { User } from 'app/common/interfaces/mainflux.interface';
 import { UsersService } from 'app/common/services/users/users.service';
 
 import { STRINGS } from 'assets/text/strings';
-
+import { environment } from '../../../../environments/environment';
```

Comment:
`environments/environment` is ok here

Reply:
doesnt work for me

Reply:
Find a solution please. As you can see we are using absolute path everywhere.